configs/warp10.json

```diff
@@ -1,10 +1,10 @@
 {
   "index_name": "warp10",
   "start_urls": [
-    "http://www.warp10.io"
+    "https://www.warp10.io"
   ],
   "stop_urls": [
-    "/tags/"
+     "/$"
```

Comment:
Using this regex will avoid crawling `/content/` pages.

We will only crawl `/doc/`page. Is it fine for you?

Reply:
You can try it out straight away (Changes are live)

Reply:
Hi, thanks, I will remove any stop url.
My initial idea was to not crawl urls with a trailing /